### ISSUE TITLE
FatalErrorException when trying to assemble chunks

### ISSUE THREAD

**Issue by zelding** [1up-lab/OneupUploaderBundle#269]
Hi, 

I get `Error: Call to a member function getPathname() on a non-object` in `Oneup/UploaderBundle/Uploader/Chunk/Storage/FilesystemStorage.php` @ 68 for `$base->getPathname()`

I suspect there is an error in my filesystem-config, but i can't find it.

My config looks like this

```
oneup_uploader:
    chunks:
        storage:
            directory:     %kernel.cache_dir%/videos/chunks/
    mappings:
        videos:
            frontend:           fineuploader
            storage:
                 directory:      %kernel.cache_dir%/videos/
```

Please help @bytehead , thank you!

**Comment by zelding** [comment]
I've found a solution to the error, it seems to be a Bug; see the pull-request below:

#270

**Comment by Sander-Toonen** [comment]
I don't know whether this is related. But ever since upgrading to PHP7 I get an error uploading large files. Apparently no EX_LOCK can be obtained on the file while reassembling the chunks.

> Uncaught PHP Exception RuntimeException: "Reassembling chunks failed." at .../vendor/oneup/uploader-bundle/Oneup/UploaderBundle/Uploader/Chunk/Storage/FilesystemStorage.php line 69

**Comment by gpaton** [comment]
@zelding I think your issue is related to the Symfony version you use. I had the same error as you with Symfony `2.7.1`. Then, I've seen the fix in issue #190 which removes what you suggest (cf. your PR #270) to fix your issue. So I've updated to Symfony `2.7.5` and it works now as it should.
